spec/features/athletes/athlete_index_spec.rb

```diff
@@ -48,5 +28,21 @@
         expect(page).to have_content(@stace.birthday.strftime("%-m/%-d/%Y"))
       end
     end
+
+    it "has a link to edit athlete" do
+      within "#athlete-#{@stace.id}" do
```

Comment:
Your use of the within block is great! It wasn't until Meg showed me during our pair project in Mod 3 that I finally recognized how much stronger it makes your testing.